frontend/src/app/shared/pipes/unit-value-transformation/unit-transformation.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { UnitValueTransformationPipe } from './unit-value-transformation.pipe';
```

Comment:
1. Der Dateiname des Tests passt nicht.
2. Die Pipe ist nicht getestet. Eine Pipe lässt sich sehr einfach testen.

**Allgemein**
- Ist es nicht notwendig, bei beiden Pipes der Wert (Zahl) und die Einheit in der Pipe zu verarbeiten:
   - `10'000 CHF` oder `10'000` oder `10 000 CHF` oder `10000` -> parse -> `10000` + `Type Unit.CHF`
   - `10000` + `Type Unit.CHF` -> format ->  `10'000 CHF`
- Könntest du nicht die Pipes zum Beispiel nicht so benennen:
   - parse-number-and-unit.pipe.ts  (Parse den eingegebener Wert)
   - format-number-and-unit.pipe.ts (Formatiere den Wert für eine schöne Ausgabe)

_Wir müssen dies miteinander nach dem Mittag besprechen._